### PR TITLE
[5.2] Add a validator helper as a shorthand for Validate::make()

### PR DESCRIPTION
**Pull request by AdenFraser**

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
:-1: for adding more helpers.

**Comment by JosephSilber** [comment]
@GrahamCampbell I love me some helpers :8ball:

File: src/Illuminate/Foundation/helpers.php [hunk 1]
```diff
@@ -761,3 +761,25 @@ function view($view = null, $data = [], $mergeData = [])
...
+     *
+     * @param  array  $data
+     * @param  array  $rules
+     * @param  array  $messages
+     * @param  array  $customAttributes
+     * @return \Illuminate\Contracts\Validation\Validator
+     */
+    function validate(array $data, array $rules, array $messages = [], array $customAttributes = [])
+    {
+        $factory = app('Illuminate\Contracts\Validation\Factory');
```
Comment (GrahamCampbell) [inline src/Illuminate/Foundation/helpers.php]: Please import this and use `::class` notation.

**Comment by AdenFraser** [comment]
Will do!

**Comment by GrahamCampbell** [comment]
Also, please rebase to avoid conflicts when importing since I recently modified this file.

**Comment by AdenFraser** [comment]
Squashed the amend in and rebased to the HEAD of 5.2, I assume that's correct?

**Comment by taylorotwell** [comment]
Seems like it should be named `validator`?

**Comment by AdenFraser** [comment]
That would make more sense actually, given the other factory helpers are defined in the same way.

**Comment by taylorotwell** [comment]
Helper functions are kept in alphabetical order.

**Comment by GrahamCampbell** [comment]
> Helper functions are kept in alphabetical order.

Ha. We probably need a StyleCI check for that, lol. The whole if statement stuff makes that sort of check slightly harder to implement, which is a shame.

**Comment by AdenFraser** [comment]
Sorting that now :)

**Comment by AdenFraser** [comment]
Hopefully thats all good! :grinning:

**Pull request merged**